app/grandchallenge/components/templatetags/base64.py

```diff
@@ -0,0 +1,11 @@
+import json
+from base64 import b64encode
+
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def b64encode_json(value):
```

Comment:
With the data put into the table or a model this filter is unnecessary.

Reply:
Deleted